packages/designto-flutter/painting/painting-box-decoration.ts

```diff
@@ -20,6 +22,7 @@ function fromBackground(b: Background): BoxDecoration {
     switch (b.type) {
       case "gradient": {
         console.error("gradient bg not ready");
```

Comment:
Was gradient on flutter never been supported? if so remove the `console.log`

Reply:
removed!